submissions/ArthurGorbenko/html5-form/index.html

```diff
@@ -99,7 +99,7 @@ <h4 class="form-header_titles form-header_info_title">
                   alt="website logo"
                   class="form-radio_choice_type_image"
                 />
-                <span><b>Website</b></span>
+                <span class='form-label-images_title'>Website</span></span>
```

Comment:
Check the number of closing `span` tags.

Reply:
Same.